src/Http/Responses/DataResponse.php

```diff
@@ -77,6 +77,13 @@ protected function getRedirect()
             throw new NotFoundHttpException;
         }
 
+        $code = 302;
+
+        if (is_array($redirect)) {
+            $code = $redirect['status'];
```

Comment:
When testing this out locally, I ran into an issue where this wasn't a string but instead a `Value` instance so the redirect couldn't work.

The tests were passing but I realised that was because they were missing the fields in the blueprint. Once I added them, the tests started to fail with the same error I'm getting in my browser.

I attempted to fix it by adding `->value()->value()` which fixes it for me locally but [still failed in GitHub Actions](https://github.com/statamic/cms/actions/runs/7885879415/job/21517927743). There's probably another way to fix it - I'll loop back to it with a fresh head at some point if you don't beat me to it.

Reply:
Got it (eventually).